sarxarray/_io.py

```diff
@@ -13,6 +14,60 @@
 # Example: https://docs.dask.org/en/stable/array-creation.html#memory-mapping
 
 
+def from_zarr(files_zarr: str | Path) -> xr.Dataset:
+    """Read a SLC stack or relabted variables from zarr files.
```

Comment:
```suggestion
    """Read a SLC stack or related variables from zarr files.
```

Reply:
It is not clear what "or related variables" means. This function does not support selecting only one variable to read from a zarr file, right? I suggest removing "or related variables".